infrastructure/terraform/orchestration.tf

```diff
@@ -88,15 +88,13 @@ module "logic_app_orchestration" {
       parameter_values = {}
     }
     videoindexer-v2 = {
-      kind         = "V2"
-      display_name = "Video Indexer (V2)"
-      description  = "Easily extract insights from your videos and quickly enrich your applications to enhance discovery and engagement. Use the Video Indexer connector to turn your videos into insights."
-      icon_uri     = "https://connectoricons-prod.azureedge.net/releases/v1.0.1654/1.0.1654.3410"
-      brand_color  = "#127B66"
-      category     = "Standard"
-      parameter_values = {
-        api_key = var.videoindexer_api_key
-      }
```

Comment:
Let's check the automated deployment, but I believe this will lead to an error in the API connection. We have to at least provide an empty string. Let's validate in the Portal.

Reply:
As expected: 
![image](https://github.com/Azure/ai-hub/assets/34542414/af7085e8-d7ef-4e26-8a8d-ea15e367ab06)
Please provide an empty string

Reply:
great point. addressed in latest commit.